pkgs/top-level/all-packages.nix

```diff
@@ -10284,6 +10284,10 @@ with pkgs;
   };
   libxcrypt-legacy = libxcrypt.override { enableHashes = "all"; };
 
+  libxeddsa = callPackage ../development/libraries/libxeddsa { };
+
+  xeddsa = python3Packages.xeddsa;
+
```

Comment:
We don't need to expose this here since it's not an application, but just a python library.

```suggestion
```